src/View/Requirements_Backend.php

```diff
@@ -791,10 +827,15 @@ public function includeInHTML($content)
         }
 
         // Add all inline JavaScript *after* including external files they might rely on
-        foreach ($this->getCustomScripts() as $script) {
+        foreach ($this->getCustomScripts() as $key => $script) {
+            // Build html attributes
+            $customHtmlAttributes = ['type' => 'application/javascript'];
+            foreach ($this->customScriptAttributes[$key] as $attrKey => $attrValue) {
+                $customHtmlAttributes[$attrKey] = $attrValue;
+            }
```

Comment:
```suggestion
            if (isset($this->customScriptAttributes[$key])) {
                foreach ($this->customScriptAttributes[$key] as $attrKey => $attrValue) {
                    $customHtmlAttributes[$attrKey] = $attrValue;
                }
            }
```
Apologies, I should have included this earlier, caused a unit test failure

Reply:
Sweet as :D, showing your refactors have been helpful, gives me a good idea how you like formatting things for future PRs. I've been following the work from Requirements::javascript before this.